sdk/src/beta9/abstractions/map.py

```diff
@@ -69,11 +69,28 @@ def stub(self) -> MapServiceStub:
     def stub(self, value: MapServiceStub):
         self._stub = value
 
-    def set(self, key: str, value: Any) -> bool:
+    def set(self, key: str, value: Any, ttl: int = 604_800) -> bool:
+        """
+        Set a key in the map.
+
+        Args:
+            key: The key to set.
+            value: The value to set.
+            ttl: The time to live for the key in seconds. Cannot be more than 7 days. Defaults to 7 days.
+
+        Raises:
+            ValueError: If the key could not be set.
+
+        Returns:
+            bool: True if the key was set, False otherwise.
+        """
```

Comment:
I think we should mention the payload size limit here

Reply:
Good call. Just updated.